Block becas

Clarissa0512 opened the pull request:
### Issues

_¿Qué se está resolviendo?_

- Poniendo el contenido de scholarship-item flexible.

### Todos

_¿Qué queda por hacer?_

- Revisar

### Áreas impactadas

_¿Qué páginas o componentes están afectados?_

- scholarship-item.html y scholars.yml

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

Clarissa0512 committed:
Merge branch 'master' of https://github.com/cosola-macile/macile.org

 Conflicts:

app/modules/teaser_news/teaser.macro.html

Clarissa0512 committed:
Agregando macro a scholarship-item.html

Aquí agrego un macro al archivo scholarship-item.html.

Clarissa0512 committed:
Añadiendo el archivo scholarship-item.html a scholarships

Aqui añado el archivo scholarship-item.html a scholarships.

Clarissa0512 committed:
Añadiendo los archivos del block becas

Aquí añado los archivos del block becas de html e yml

Clarissa0512 committed:
Añadiendo loops a los archivos de scholarship-item.html

Aquí añado loops a scholarship-item.html.

Clarissa0512 committed:
Añadiendo cambios en scholarship-item

misscs commented on app/modules/scholarship/scholarship-item.html:
```diff
@@ -1,12 +1,29 @@
+  {% if scholars.pre_college %}
     <figure class="scholarship-figure">
-      <img src="/assets/img/team-foto.jpg" alt="imagen of member">
+      <img scr="{{ scholars.photo }}">
```
No olvides el `alt`! Siempre debemos utilizar este atributo cuando usamos `<img>` etiqueta

misscs commented:
@Clarissa0512 Buen esfuerzo 💪 Cuando empezamos clase de nuevo, yo revisare este PR con el groupo completo.

misscs committed:
Corregir errores

- Nombres de macros no pueden usar guión
- Cerrar el macro

misscs committed:
Imprimir nombre

misscs committed:
Anadir llave para idioma

misscs committed:
Creando la structura con loops para imprimir el contenido

Clarissa0512 committed:
Conflicts:
	app/modules/scholarship/scholarship_item.html
	app/views/scholarships/index.html

Clarissa0512 committed:
Resolviendo conflicto en el archivo scholarships_item.html

Clarissa0512 committed:
Removiendo el import del archivo index en scholarships

Aquí remuevo el import en el archivo index de scholarships.

Clarissa0512 committed:
Añadiendo year start en scholarships.

Aqui añado year-start en el loop para imprimirlo en la información sobre los becados

Clarissa0512 committed:
Añadiendo year end en scholarships

Aqui añado year-end en el loop para imprimirlo en la información sobre los becados.

Clarissa0512 committed:
Añadiendo location en el index de scholarships

Aqui añado location en el loop para imprimirlo en la información sobre los becados.

Clarissa0512 committed:
Añadiendo year start para college en scholarships.

Aqui añado year-start en el loop para imprimirlo en la información sobre los becados en college.

Clarissa0512 committed:
Añadiendo year end para college en scholarships.

Aqui añado year-end en el loop para imprimirlo en la información sobre los becados en college.

Clarissa0512 committed:
Añadiendo location para college en scholarships.

Aqui añado location en el loop para imprimirlo en la información sobre los becados en college.

Clarissa0512 committed:
Cambiando las etiquetas del contenido del index de scholarships.

Aqui cambio las etiquetas que estaban con h4 a p.

Clarissa0512 committed:
Cambiando nombre de los archivos en scholarships.

Aqui cambiamos la carpeta scholarships a scholar y añadimos un macro.

Clarissa0512 committed:
Añadiendo el macro de scholar e insertando el header

Aquí ponemos el contenido con un macro e insertamos el header para la pagina scholarships.

Clarissa0512 committed:
Cambiando los nombres de las clases de scholarship a scholar.

Aquí cambiamos el nombre de las clases de scholarship a scholar.

Clarissa0512 committed:
Cambiando los nombre de las clases y el nombre del archivo css
Aqui cambio los nombres de las clases y del archivo css de scholarship a scholar.

Clarissa0512 committed:
Removiendo el archivo de scholarship.css

Clarissa0512 committed:
Añadiendo el nuevo nombre del scholar.css en site.css

Aqui añado el nuevo nombre del archivo scholar.css a site.css

Clarissa0512 committed:
Merge branch 'master' into block-becas

 Conflicts:
	app/assets/styles/site.css
	app/views/scholarships/index.html

Clarissa0512 committed:
Removiendo include por loop

Aqui removemos el include y le ponemos un loop.
